From 46c04d4ce8c7f11cea906ff61f0875e108d6b609 Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Tue, 24 Dec 2019 02:55:38 +0000
Subject: [PATCH 242/245] Revert "arm64: dts: ls1088a: Added dcfg node in
 ls1088a dtsi"

This reverts commit 88b64bb1aaaa1054144acbfa529a3bc3b8004600.
There already exists dcfg node, duplicated one should be deleted.

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
index be2381843cc9..ca3cadff65a9 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
@@ -178,12 +178,6 @@
 			clocks = <&sysclk>;
 		};
 
-		dcfg: dcfg@1e00000 {
-			compatible = "fsl,ls1088a-dcfg", "syscon";
-			reg = <0x0 0x1e00000 0x0 0x10000>;
-			little-endian;
-		};
-
 		tmu: tmu@1f80000 {
 			compatible = "fsl,qoriq-tmu";
 			reg = <0x0 0x1f80000 0x0 0x10000>;
-- 
2.17.1

